CONTRIBUTING.md

```diff
@@ -42,7 +42,19 @@ uv sync
 
 This installs httptap in editable mode with all development dependencies.
 
-3. **Verify installation**
+3. **Set up pre-commit hooks (recommended)**
+
+```bash
+# Install pre-commit
+pip install pre-commit
```

Comment:
I suggest adding a pre-commit to requirements (pyproject.toml) and using commands like `uv run pre-commit run --all-files'.
You can skip this step.